### PR TITLE
[Ready to Merge] Vs docker tools config

### PR DESCRIPTION
**Pull request by ghogen**
## Summary

Incorporate product team feedback on the VS Docker Tools container eBook topic.

### CODE REVIEW

**Comment by ghogen** [comment]
This is a work in progress for now, don't merge yet.

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md [hunk 1]
```diff
@@ -70,12 +70,15 @@ Figure 4-30: Docker Tools Options
...
+| Automatically start containers in background | On | Docker Compose | Again for increased performance, Visual Studio creates a container with volume mounts ready for when you build and run your container. If you want to control when your container is created, turn this off. |
+| Automatically kill containers on solution close | On | Docker Compose | Turn this off if you would like containers for your solution to continue to run after closing the solution or closing Visual Studio. |
+| Do not prompt for trusting localhost SSL certificate | Off | ASP.NET Core 2.1 projects | If the localhost SSL certificate is not trusted, Visual Studio will prompt every time you run your project, unless this checkbox is checked. |
+
+> [!WARNING]
+> If the localhost SSL certificate is not trusted, and you check the box to suppress prompting, then your project can't run. In that case, uncheck the **Do not prompt** checkbox, run your project, indicate trust at the prompt.
```
Comment (rpetrusha) [inline docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md]: an "and" should separate the comma and "indicate"

**Review by rpetrusha — CHANGES_REQUESTED** [review]
This looks good, @ghogen. I've just left one minor comment.

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md (hunk 1 above)
Comment (ghogen) [reply docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md]: Thanks

**Review by rpetrusha — APPROVED** [review]

**Comment by ghogen** [comment]
Waiting on PM review.

**Comment by ghogen** [comment]
I incorporated review, so this is good now.  @mairaw I don't have permissions to change labels, but I incorporated review feedback, so this is ready to merge.

**Comment by ghogen** [comment]
@rpetrusha This is good to merge now.

**Pull request merged**